### DISCUSSION TITLE
Cal Hacks 10.0 FAQ

### DISCUSSION THREAD

**Discussion by MyoungXUE** [zepp-health/zeppos-forum#262] [Blog]
# FAQ

## Apple Silicon Mac Environment Setup

If you are using an Apple Silcon series Mac, please refer to this procedure to install the environment

**Installation script env-setup-script may have problems, please manually install the environment**

1. Enable System Extensions or Kernel Extension on Apple Silicon Mac
https://mediaseal.fortiumtech.com/support/solutions/articles/30000047239-how-to-enable-system-extensions-on-your-m1-m2-mac-

2. Download this app for M chip Mac https://tunnelblick.net/downloads.html, and finish Tap/Tun install
https://docs.zepp.com/docs/guides/tools/simulator/setup/#2-configuring-the-virtual-network-card-1

3. Install Node.js and zeus-cli

https://docs.zepp.com/docs/guides/quick-start/environment/#manual-installation
https://docs.zepp.com/docs/guides/tools/cli/#installing-the-zeus-cli

---

If you have already executed env-setup-script and the environment does not work, then please re-do the second step above, uninstall Tap/Tun and then reinstall Tap/Tun using Tunnelblick.

## `zeus dev` does not work

If the emulator does not show the Mini Program installed after executing `zeus dev`, try using the following command

`zeus config set simulator_host=127.0.0.1`

https://docs.zepp.com/docs/guides/tools/cli/#zeus-config-configuration-related

**Comment by zeppos-hx** [comment]
In M chips Mac, if you have setup by using env-setup-script and tap tun still not working, you may need delete these old extensions and plist files, use the cmd:

`sudo rm -rf /Library/Extensions/tun.kext`
`sudo rm -rf /Library/Extensions/tap.kext`

`sudo rm -f /Library/LaunchDaemons/*tun.plist`
`sudo rm -f /Library/LaunchDaemons/*tap.plist`

And then install the tun tap driver via Tunnelblick app again.
